randovania/games/samus_returns/json_data/Area 3 - Exterior.txt

```diff
@@ -731,35 +731,13 @@ Extra - asset_id: collision_camera_038
   * Extra - actor_name: Door015
   * Extra - actor_type: doorpowerpower
   > Event - Gamma Metroid
-      All of the following:
-          Defeat Gamma Metroid and Lay Any Bomb
-          Walljump (Beginner) or Climb Rooms Vertically (High Jump)
+      Climb Rooms Vertically (High Jump) and Defeat Gamma Metroid and Lay Any Bomb
 
 > Event - Gamma Metroid; Heals? False
   * Layers: default
   * Event Area 3 (Exterior) - Gamma Metroid
   > Door to Gamma Shaft
-      Any of the following:
-          Spider Ball and Lay Any Bomb
-          All of the following:
-              Lay Power Bomb
-              Space Jump or Single-wall Wall Jump (Intermediate)
-          All of the following:
-              Lay Bomb
-              Any of the following:
-                  Gravity Suit and Infinite Bomb Jump (Beginner)
-                  All of the following:
-                      Infinite Bomb Jump (Intermediate)
-                      Spring Ball or Mid-Air Morph (Intermediate)
-                  Mid-Air Morph (Intermediate) and Single-wall Wall Jump (Intermediate)
-  > Start Point
-      Trivial
-
-> Start Point; Heals? False; Default Node
```

Comment:
Was the start point removed?

Reply:
I removed some start points because we're now out of place or unnecessary.  I'll add custom spawn points later that make sense.